CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED][unreleased]
 
+## [1.6.1][1.6.1]
+### Added
+- Addition of new IFSCs
```

Comment:
there are even more IFSCs you can remove these ^ line for now
Add this line as well
metadata changes in this release